src/compositions/mumble/mumble.tsx

```diff
@@ -30,10 +33,23 @@ export const Mumble: FC<MumbleProps> = ({
   timestamp,
   content,
   actions,
+  hideBorder = false,
+  showUserInline = false,
 }) => {
   return (
-    <div className="relative bg-white rounded-2xl hover:outline-neutral-200 hover:outline-2 shadow-sm pt-1 pr-12 pb-8 pl-12">
-      <div className="absolute top-6 -left-6 z-10">
+    <div
+      className={twMerge(
+        "relative bg-white rounded-2xl pt-1 pb-8",
+        hideBorder ? "" : "shadow-sm hover:outline-neutral-200 hover:outline-2",
```

Comment:
Not sure, but could we use this syntax here and below?

`hideBorder && "shadow-sm hover:outline-neutral-200 hover:outline-2"`